lua/telescope/make_entry.lua

```diff
@@ -331,10 +362,8 @@ function make_entry.gen_from_symbols(opts)
     return displayer {
       {entry.lnum .. ":" .. entry.col, "LineNr"},
       entry.symbol_name,
-      {"[", "TelescopeBorder"},
-      {entry.symbol_type, type_highlight[entry.symbol_type], type_highlight[entry.symbol_type]},
-      {"]", "TelescopeBorder"},
-      filename,
+      {entry.symbol_type:lower(), type_highlight[entry.symbol_type], type_highlight[entry.symbol_type]},
```

Comment:
Do we no longer have borders? Or do i need to add them back in my PR.

Reply:
I'm not going to use them for the LSP/TS finders. Too visually noisy.
I do intend to use them in the registers finder and others though